joomla/joomla-cms
# [4.0] Fixed the check-in option on Update Sites and clean language files

Pull Request for Issue #26166

### Summary of Changes
This PR fixes the Check-in button on the update sites page.

### Testing Instructions
1. Go to System
2. Click on Update -> Update Sites
3. Select an item from the list
4. Click on Check-in
5. Notice nothing happens
6. Apply PR
7. Select an item from the list
8. Click on Check-in and notice you get a message
9. Repeat the process with multiple items and see you get the correct message

Further the language files are cleaned up.


### Expected result
Items are checked-in


### Actual result
Nothing happens


### Documentation Changes Required
None

@brianteeman
